### PR TITLE
[dynamo][dashboard] Run 2 iterations for the correctness runs

### PR DESCRIPTION
**Pull request by anijain2305**

### CODE REVIEW

File: benchmarks/dynamo/huggingface.py [hunk 1]
```diff
@@ -437,6 +437,7 @@ def forward_pass(self, mod, inputs, collect_outputs=True):
     def forward_and_backward_pass(self, mod, inputs, collect_outputs=True):
         cloned_inputs = clone_inputs(inputs)
         mod.zero_grad(True)
+        self.optimizer_zero_grad()
```
Comment (soumith) [inline benchmarks/dynamo/huggingface.py:440]: you dont need `mode.zero_grad(True)` and `self.optimizer_zero_grad()`. Only one of them is sufficient I believe.

File: benchmarks/dynamo/huggingface.py [hunk 2]
```diff
@@ -437,6 +437,7 @@ def forward_pass(self, mod, inputs, collect_outputs=True):
     def forward_and_backward_pass(self, mod, inputs, collect_outputs=True):
         cloned_inputs = clone_inputs(inputs)
```
Comment (soumith) [inline benchmarks/dynamo/huggingface.py:438]: curious, why do we clone inputs here?

**Review by soumith — APPROVED** [review]

File: benchmarks/dynamo/huggingface.py (hunk 1 above)
Comment (anijain2305) [reply benchmarks/dynamo/huggingface.py:440]: Cool, will keep optimizer.zero_grad as it seems to be more popular.

File: benchmarks/dynamo/huggingface.py (hunk 2 above)
Comment (anijain2305) [reply benchmarks/dynamo/huggingface.py:438]: this is to keep the original inputs unmodified, when the model mutates the inputs.

**Pull request closed without merging**